src/packages/containers/App.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import { TypeHere, Preview, FormType } from '../index';
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <TypeHere
+          schemas="weburl,post,image"
```

Comment:
Pass in schemas here or higher up.

NB: `Schemas` are used in combination with `Rules` and `TypeHere` to help determine which `FormType` to render. Example, if I set `schemas` only with `post` and User types `hello kitty` I would expect the `Rules` to return `PostFormType` and render.